internal/worker/api/errors.go

```diff
@@ -148,14 +148,17 @@ func apiErrorFromEchoError(echoError *echo.HTTPError) ServiceErrorCode {
 
 // Convert an echo error into an AOC compliant one so we send a correct json error response
 func HTTPErrorHandler(echoError error, c echo.Context) {
-	doResponse := func(code ServiceErrorCode, c echo.Context) {
+	doResponse := func(code ServiceErrorCode, c echo.Context, internal error) {
 		if !c.Response().Committed {
 			var err error
 			sec := find(code)
 			apiErr := APIError(code, sec, c)
 
 			if sec.httpStatus == http.StatusInternalServerError {
-				c.Logger().Errorf("Internal server error. Code: %s, OperationId: %s",
+				c.Logger().Errorf("Internal server error. Internal: %s, Code: %s, OperationId: %s",
+					internal, apiErr.Code, apiErr.OperationId)
+			} else {
+				c.Logger().Errorf("%s, Code: %s, OperationId: %s", internal,
```

Comment:
Not sure if this is Error() worthy; in the client yes, but in the server this can be a 4XX as well. In the cloudapi/v2 nothing gets logged if it's not a 500.

Reply:
Makes sense, since this is for everything, including 4xx, I'll make it Info.